cmd/dcrctl/config.go

```diff
@@ -280,50 +285,46 @@ func loadConfig() (*config, []string, error) {
 // For this it tries to read the dcrd config file at its default path, and extract
 // the RPC user and password from it.
 func createDefaultConfigFile(destinationPath string) error {
-	// Create the destination directory if it does not exists.
-	err := os.MkdirAll(filepath.Dir(destinationPath), 0700)
-	if err != nil {
-		return err
-	}
+	// Create the destination directory if it does not exists
+	os.MkdirAll(filepath.Dir(destinationPath), 0700)
```

Comment:
That was part of making it the same as btcd.  I'll add the error back and add it to btcd as well.